physionet-django/project/templates/project/published_project.html

```diff
@@ -324,9 +324,15 @@ <h5 class="card-header">Versions</h5>
   <h2 id="files">Files</h2>
   {% if project.allow_file_downloads %}
   {% if project.deprecated_files %}
-    <div class="alert alert-danger col-md-8" role="alert">
-      The files for this project are no longer available.
-    </div>
+    {% if project.is_latest_version %}
```

Comment:
As the div is used in both cases, it might be a bit cleaner (avoids duplication) to move this outside the if clause? e.g. 

```
<div class="alert alert-danger col-md-8" role="alert">
  {% if project.is_latest_version %}
      The files for this project are no longer available.
  {% else %}
      The files for this version of the project ({{ project.version }}) are no longer available. The latest version of this project is <a href="{% url 'published_project' latest_version.slug latest_version.version %}" target="_blank">{{ latest_version.version }}</a>
  {% endif %}
</div>
```

Reply:
@tompollard thanks for the suggestion. I did think about that also. When searching online earlier to see if this was a good idea, I got confused by a post that discussed putting a conditional statement inside the tag itself - which isn't what we do here. I should have just tried it, because it works just fine.